cmd/cluster/resize/infra_node.go

```diff
@@ -536,3 +543,30 @@ func (r *Resize) nodesMatchExpectedCount(ctx context.Context, labelSelector labe
 
 	return false, nil
 }
+
+// having an error when beeing in a rety loop, should not be handled as an error, and we should just disaply it and continue
+// in case we have a function that return a bool statis and an error, we can use following helper
+// f beeing a function returning (bool, error), replace
+//
+//	return f(...)
+//
+// by
+//
+//	return skipError(wrapResult(f(...)), "message to context the error"
+//
+// and then the return will always have error set to nil, but a continuing message will be dispalyed in case of error
+type result struct {
+	condition bool
+	err       error
+}
+
+func wrapResult(condition bool, err error) result {
+	return result{condition, err}
+}
+
+func skipError(res result, msg string) (bool, error) {
+	if res.err != nil {
+		log.Printf("%s, continuing to wait: %s", msg, res.err)
+	}
+	return res.condition, nil
+}
```

Comment:
Would it make sense to combine `wrapResult` and `skipError`, if the recommendation is to always call it like 

```golang
return skipError(wrapResult(f(...)), "message to context the error"
```

Reply:
No, this is not really possible as goal of wrapResult is to convert the multi value return of f function in a struct in order to pass it to skipError, that takes another string argument.
Now, if there is better solution, I would be happy to learn, because indeed I do not like it much ;)